apollo-backend/src/main/java/io/logz/apollo/controllers/DeploymentController.java

```diff
@@ -164,4 +166,22 @@ public void cancelDeployment(int id, Req req) {
             lockService.releaseLock(lockName);
         }
     }
+
+    @LoggedIn
+    @POST("/deployment-history")
+    public void fetchPaginatedDeploymentHistory(Boolean descending, int pageNumber, int pageSize,  String searchTerm, Req req) {
+
+        DeploymentHistory deploymentHistory = new DeploymentHistory();
+        int start = (pageNumber - 1) * pageSize;
```

Comment:
Why not to call this parameter "offset", as the name of the parameter in filterDeploymentHistoryDetails()?
And maybe extract the part of - "(pageNumber - 1) * pageSize;" to a method with a proper name; that way it will be clear what and why you are doing it :)

Reply:
I named it "pageInitIndex" to keep it close to the "pageNumber" term (that was set by api guidelines), and also I feel it is more self-explanatory then "offset". Maybe I'm wrong, let me know..

Reply:
If the name is according to API guidelines then they know better than me ;)